plugins/woocommerce-admin/client/dashboard/components/connect/index.js

```diff
@@ -47,14 +47,14 @@ export class Connect extends Component {
 				if ( onConnect ) {
 					onConnect();
 				}
-				window.location = jetpackConnectUrl;
+				window.location = jetpackAuthUrl;
 			}
 		);
 	}
 
 	render() {
 		const {
-			hasErrors,
+			error,
```

Comment:
This is a drive-by fix. I noticed there are no references to a prop called `hasErrors` here or where this `<Connect>` component is used. I removed this prop and replaced it with a check on the `error` prop. Tested and it works.